doc/development/guide/pullrequests.rst

```diff
@@ -1,74 +1,58 @@
-  running ``make docs``. See our :doc:`documentation guidelines <documentation>` for more details.
+Once you create the PR, you can also view the status of the *Continuous Integration* (CI) checks for things like tests, code quality, and documentation rendering.
 
-* **Ensure that the test suite passes**, by running ``make test``.
+Continuous Integration Checks
+-----------------------------
 
-* **Make sure the modified code in the pull request conforms to the PEP8 coding standard.**
+Once a PR is opened on the repository, GitHub runs a series of checks on the
+branch.  All tests need to pass (green check) for the PR to get approved and
+merged.
 
-  The PennyLane source code conforms to `PEP8 standards <https://www.python.org/dev/peps/pep-0008/>`_.
-  Before submitting the PR, you can autoformat your code changes using the
-  `Black <https://github.com/psf/black>`_ Python autoformatter, with max-line length set to 100:
+* **Formatting check**: We use `black <https://black.readthedocs.io/en/stable/>`_ to autoformat our code to `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ standards. You should run ``black`` locally on changed files, as the CI check just determines whether or not ``black`` would change anything. Black installs with ``pip``. We run black on both the source folder ``pennylane`` and the ``tests`` folder. We format with a line length of 100. From the PennyLane's root folder, you can run:
 
   .. code-block:: bash
 
-      black -l 100 pennylane/path/to/modified/file.py
+      black -l 100 pennylane tests
 
-  You should also apply Black to any tests you add:
+* **Code factor**:  `Code factor <https://www.codefactor.io/>`_ checks several common code quality characteristics. 
 
-  .. code-block:: bash
-
-      black -l 100 tests/path/to/modified/test.py
+* **Tests**: Github Actions runs the core tests and device tests for a series of different Python versions and interface installations.
 
-  We check all of our code against `Pylint <https://www.pylint.org/>`_ for errors.
-  To lint modified files, simply ``pip install pylint``, and then from the source code
-  directory, run
+* **Code Coverage**: `codecov <https://app.codecov.io/gh/PennyLaneAI/pennylane/>`_ determines if all lines are executed during the automated testing and tells you which ones are not. We want all lines to be covered, but that is insufficient to determine if the testing is complete enough. That takes thought and thinking through the test design, to ensure that all edge cases are being tested.
 
-  .. code-block:: bash
+* **Documentation**: The "Documentation check" determines if there are any issues with the build, but you also need to inspect the relevant pages in the website build visually. You can access the temporary website by clicking on ``Details`` for ``docs/readthedocs.org:pennylane``, as shown in the following image.
 
-      pylint pennylane/path/to/modified/file.py
+.. image:: view_doc_build.png
+    :width: 400px
+    :align: center
```

Comment:
It might be worth making the image a little larger, or turning off dark mode before screenshotting, to make it easier to read :)